### PR TITLE
react1 week1 homework

### PR DESCRIPTION
**Pull request by Vahablotfi**

### CODE REVIEW

**Review by BikramRawat — COMMENTED** [review]
Great job, but one thing i can suggest is, just push the required files only, fx. no need to push gitignore, .json, index.html, test files any many more other files as you can see under the files changed section that you have pushed. And also i suggest to clean up the code, remove unnecessary codes that came automatically as boilerplates  fx: on the index.js i cleaned up , see under the comment section:)

File: react/react1/week1/week1hw-simple-list/src/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (BikramRawat) [inline react/react1/week1/week1hw-simple-list/src/index.js:17]: Cleanup the code : fx you can just write this for now:

import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';

const rootElement = document.getElementById("root");
ReactDOM.render(`<App />`, rootElement);

File: react/react1/week1/week1hw-simple-list/src/setupTests.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (BikramRawat) [inline react/react1/week1/week1hw-simple-list/src/setupTests.js:5]: Just push the files that you have worked on, no need to push these all files fx json files, test files, images that you didnot uploaded, gitignore files, and many other

**Review by BikramRawat — COMMENTED** [review]
👍

File: react/react1/week1/week1hw-simple-list/src/index.js (hunk 1 above)
Comment (Vahablotfi) [reply react/react1/week1/week1hw-simple-list/src/index.js:17]: Thanks for comments , :)

**Pull request merged**